cmake/external/extensions.cmake

```diff
@@ -35,11 +35,22 @@ if (onnxruntime_WEBASSEMBLY_DEFAULT_EXTENSION_FLAGS)
   set(OCOS_ENABLE_VISION OFF CACHE INTERNAL "")
 endif()
 
-# when onnxruntime-extensions is not a subdirectory of onnxruntime,
-# output binary directory must be explicitly specified.
-# and the output binary path is the same as CMake FetchContent pattern
-add_subdirectory(${onnxruntime_EXTENSIONS_PATH} ${CMAKE_BINARY_DIR}/_deps/extensions-subbuild EXCLUDE_FROM_ALL)
+# onnxruntime-extensions
+if (NOT onnxruntime_EXTENSIONS_OVERRIDDEN)
```

Comment:
is this flag necessary? can we just check whether onnxruntime_EXTENSIONS_PATH is defined here?

Reply:
The path [is always defined](https://github.com/microsoft/onnxruntime/blob/6480f665fe4875cf0238bde9dab64f0497f62200/tools/ci_build/build.py#L1233) to point to the path of extensions

Reply:
Lets get this PR merged first and than create a ticket for this new feature.